dynamicio/mixins.py

```diff
@@ -716,7 +756,6 @@ def _read_database(session: SqlAlchemySession, query: Union[str, Query], **optio
             query = query.with_session(session).statement
         return pd.read_sql(sql=query, con=session.get_bind(), **options)
 
-    @allow_options({"model"})
```

Comment:
Nicely caught, but this needs to also be updated here: https://github.com/VorTECHsa/dynamicio/blob/master/dynamicio/mixins.py#L700 to 
```python
@allow_options(pd.read_sql)
```